ir/src/main/scala/org/scalajs/core/ir/Serializers.scala

```diff
@@ -799,8 +858,24 @@ object Serializers {
             result2
           }
         case TagPropertyDef =>
-          PropertyDef(readPropertyName(), readTree(),
-              readTree().asInstanceOf[ParamDef], readTree())
+          val name = readPropertyName()
+          val getterBody = readOptTree()
+          val setterArgAndBody = if (useHacks068) {
+            for {
+              setterArg <- readTreeAllowEmpty().map(_.asInstanceOf[ParamDef])
```

Comment:
`setterArg` could not be `EmptyTree` in the old format (as is clearly seen by the existing `.asInstanceOf[ParamDef]`) so you should use `readTree()` here.